### PR TITLE
remove old log service

### PR DESCRIPTION
**Pull request by zhiweiyin318**
issue: https://issues.redhat.com/browse/ACM-10784

### CODE REVIEW

File: pkg/klusterlet/clusterinfo/controller_test.go [hunk 1]
```diff
@@ -220,7 +213,17 @@ func TestFailedClusterInfoReconcile(t *testing.T) {
 		t.Errorf("failed get updated clusterinfo ")
 	}
 
-	if meta.IsStatusConditionTrue(updatedClusterInfo.Status.Conditions, clusterv1beta1.ManagedClusterInfoSynced) {
+	if !meta.IsStatusConditionTrue(updatedClusterInfo.Status.Conditions, clusterv1beta1.ManagedClusterInfoSynced) {
```
Comment (zhujian7) [inline pkg/klusterlet/clusterinfo/controller_test.go:216]: It is odd we change `meta.IsStatusConditionTrue` to `!meta.IsStatusConditionTrue` to pass the test?

Comment (zhiweiyin318) [reply pkg/klusterlet/clusterinfo/controller_test.go:216]: the `TestFailedClusterInfoReconcile` is to test failed synced case. so `!meta.IsStatusConditionTrue` is expected.
the previous is incorrect.

**Comment by zhujian7** [comment]
LGTM

**Pull request merged**